site/themes/arangodb-docs-theme/layouts/shortcodes/embed-svg.html

```
@@ -0,0 +1,12 @@
+{{ $svgName := .Get 0 -}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)